robot-diary-web/my_index.html

```diff
@@ -7,5 +7,24 @@
     <header>
         header
     </header>
+    <section>
+        <main>
```

Comment:
the indentation was 2 spaces, but here is 4 spaces.
keep it consistent. don't mix it.